app/src/main/java/io/apicurio/designer/storage/h2/H2DesignerSqlStatements.java

```diff
@@ -16,23 +16,32 @@
 
 package io.apicurio.designer.storage.h2;
 
+import io.apicurio.common.apps.storage.exceptions.StorageException;
+import io.apicurio.common.apps.storage.sql.jdbi.Handle;
 import io.apicurio.common.apps.storage.sql.jdbi.query.Update;
 import io.apicurio.common.apps.storage.sql.jdbi.query.UpdateImpl;
 import io.apicurio.designer.spi.storage.DesignerSqlStatements;
 import io.apicurio.designer.storage.common.AbstractDesignerSqlStatements;
-import io.quarkus.arc.DefaultBean;
 
 import java.sql.SQLException;
 import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Named;
 
 /**
  * @author eric.wittmann@gmail.com
  * @author Jakub Senko <em>m@jsenko.net</em>
  */
 @ApplicationScoped
-@DefaultBean
+@Named("h2sqlStatements")
```

Comment:
I think these do not have to be beans, so we can just create them in the producer method, avoiding `@Named`.

Reply:
You're right, I did this part without looking at how I did it for Registry, and these two classes are not beans for sure. Removing that, thanks!